otwcode/otwarchive
# AO3-6157 Update mime-types and rest-client gems

## Issue

https://otwarchive.atlassian.net/browse/AO3-6157

## Purpose

Updates the mime-types gem to silence [an error](https://github.com/mime-types/ruby-mime-types/pull/146), and update rest-client because it depended on an older version of mime-types.

## Testing Instructions

My guess is going to be make sure tests run, the site loads, and icon uploads work.
